### PR TITLE
Solution To New Scheme Problem of codechef in C++

### PR DESCRIPTION
**Pull request by Kuljeet-123**
**Fixes issue:** #4563 

**Changes:**

added solution to [NEWSCH](https://www.codechef.com/problems/NEWSCH) in C++.

### CODE REVIEW

File: code/online_challenges/src/codechef/NEWSCH/NEWSCH.cpp [hunk 1]
```diff
@@ -0,0 +1,50 @@
+#include <iostream>
+
+using namespace std;
+
+int tt,n,m,old,it,i,j,k,x[222],y[222],u[222];
```
Comment (arnavb) [inline code/online_challenges/src/codechef/NEWSCH/NEWSCH.cpp]: Please declare these variables close to where they arre initially used.

File: code/online_challenges/src/codechef/NEWSCH/NEWSCH.cpp [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+bool g[222][222],q;
```
Comment (arnavb) [inline code/online_challenges/src/codechef/NEWSCH/NEWSCH.cpp]: Declare all of these in main.

File: code/online_challenges/src/codechef/NEWSCH/NEWSCH.cpp [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+          if (!g[j][k]) { q=false; break; 
+        }
+      }
+        if (q)
+        {
+          u[i]=it++;
+          for (j=1; j<=n; j++) if (g[i][j]) g[i][j]=g[j][i]=0;
+          break;
+        }
+      }
```
Comment (arnavb) [inline code/online_challenges/src/codechef/NEWSCH/NEWSCH.cpp]: Reforrmat this file using clang format and 4 spaces. Also in all for loops use pre-increment.

File: code/online_challenges/src/codechef/NEWSCH/NEWSCH.cpp [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    for (i=0; i<m; i++) 
+    {
+      scanf("%d%d",&x[i],&y[i]);
+      g[x[i]][y[i]]=true;
+      g[y[i]][x[i]]=true;
+    }
+    for (it=1; it<=n; )
+    {
+      old=it;
+      for (i=1; i<=n; i++) if (u[i]==0)
```
Comment (arnavb) [inline code/online_challenges/src/codechef/NEWSCH/NEWSCH.cpp]: In C++ loop variables can be declared in the loop directly.

**Comment by Kuljeet-123** [comment]
@arnavb Please review it and merge it also.

File: code/online_challenges/src/codechef/NEWSCH/NEWSCH.cpp [hunk 5]
```diff
@@ -0,0 +1,52 @@
+#include <iostream>
+
+#include <cstring>
+
+using namespace std;
```
Comment (arnavb) [inline code/online_challenges/src/codechef/NEWSCH/NEWSCH.cpp:5]: What about your other codechef PR. It seems to also have these changes. The STRWN one.

**Comment by Kuljeet-123** [comment]
@arnavb Due to conflicts , I am closing this PR and will raise a new PR. Because I  forked the repo again

**Pull request closed without merging**